manywheel/Dockerfile_cuda_aarch64

```diff
@@ -75,9 +75,10 @@ ADD ./common/install_magma.sh install_magma.sh
 RUN bash ./install_magma.sh ${BASE_CUDA_VERSION} && rm install_magma.sh
 
 FROM base as openblas
+ARG BASE_CUDA_VERSION
 # Install openblas
```

Comment:
Do we even need BASE_CUDA_VERSION? 
We could just 
RUN bash ./install_openblas.sh CUDA_ONLY 

and then in install_openblas.sh, check if $1 is present?

Reply:
For ARM CPU build, is BASE_CUDA_VERSION empty or "cpu"?

Reply:
It may not be worth it to add an extra variable CUDA_ONLY as we could we reuse the BASE_CUDA_VERSION. For CPU build, it will be empty (unset). See https://github.com/pytorch/builder/blob/main/manywheel/build_docker.sh#L36.

Reply:
It is not a variable, just an argument, string. 

The purpose of BASE_CUDA_VERSION is to just supply a string, right? 

Either way it works.

Reply:
BASE_CUDA_VERSION will be supplied only when CUDA arm build is running, so I think it might be reasonable to use.